xmlvalidate/activity.go

```diff
@@ -1,43 +1,55 @@
 package xmlvalidate
 
 import (
-	"bytes"
 	"context"
 	"fmt"
 	"os"
-	"os/exec"
-	"path/filepath"
 
 	"go.artefactual.dev/tools/temporal"
 )
 
 const Name = "xml-validate"
 
+// An XSDValidator validates the XML document at xmlPath against the XSD schema
+// at xsdPath.
+type XSDValidator interface {
+	Validate(ctx context.Context, xmlPath, xsdPath string) (string, error)
```

Comment:
Does `xmlPath` need to be specified to be a string?

Reply:
`xmlPath` is specified as a string type, Go allows declaring multiple parameters with the same type using a single type statement: https://go.dev/tour/basics/5 :)